asb-ballerina/Module.md

```diff
@@ -1,473 +1,262 @@
+## Quickstart
 
-# Quickstart(s):
+To use the Azure Service Bus connector in your Ballerina application, update the .bal file as follows:
 
-## Send and Receive Messages from the Azure Service Bus Queue
-
-This is the simplest scenario to send and receive messages from an Azure Service Bus queue. You need to obtain 
-a connection string of the name space and an entity path name of the queue you want to send and receive messages from. 
-
-### Step 1: Import the Azure Service Bus Ballerina Library
-First, import the ballerinax/asb module into the Ballerina project.
+### Step 1: Import connector
+First, import the `ballerinax/asb` module into the Ballerina project.
 ```ballerina
     import ballerinax/asb as asb;
 ```
 
-### Step 2: Initialize the Azure Service Bus Client Configuration
-You can now make the connection configuration using the connection string.
+### Step 2: Create a new connector instance
+Create an `asb:AsbConnectionConfiguration` with the connection string obtained and initialize the connector with it.
 ```ballerina
     asb:AsbConnectionConfiguration config = {
        connectionString: <CONNECTION_STRING>
     };
-```
 
-### Step 3: Create an Azure Service Bus Client using the connection configuration
-You can now make an Azure service bus client using the connection configuration.
-```ballerina
     asb:AsbClient asbClient = new (config);
 ```
 
-### Step 4: Create a Queue Sender using the Azure service bus client
-You can now make a sender connection using the Azure service bus client. Provide the `queueName` as a parameter.
-```ballerina
-    handle queueSender = checkpanic asbClient->createQueueSender(queueName);
-```
+### Step 3: Invoke connector operation
+1. Now you can use the operations available within the connector. Note that they are in the form of remote operations.
 
-### Step 5 : Create a Queue Receiver using the Azure service bus client
-You can now make a receiver connection using the connection configuration. Provide the `queueName` as a parameter. 
-Optionally you can provide the receive mode which is PEEKLOCK by default.
-```ballerina
-    handle queueReceiver = checkpanic asbClient->createQueueReceiver(queueName, asb:RECEIVEANDDELETE);
-```
+    Following is an example on how to send messages to the Azure Service Bus using the connector. You need to create a 
+    `queueSender` before sending a message.
 
-### Step 6: Initialize the Input Values
-Initialize the message to be sent with message body, optional parameters and properties.
-```ballerina
-    // Input values
-    string stringContent = "This is My Message Body"; 
-    byte[] byteContent = stringContent.toBytes();
-    map<string> properties = {a: "propertyValue1", b: "propertyValue2"};
-    int timeToLive = 60; // In seconds
-    int serverWaitTime = 60; // In seconds
-
-    asb:ApplicationProperties applicationProperties = {
-        properties: {a: "propertyValue1", b: "propertyValue2"}
-    };
+    Send a message to the Azure Service Bus
```

Comment:
Do we need this, as this is explained above?

Reply:
This is followed in the common format. 
https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit?usp=sharing